tests/test_coordsys.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import numpy as np
+import amrex
+
+def test_coordsys():
+    pass
```

Comment:
Test missing.
Idea: construct, print, assert its value, re-assign value and assert its value again.